src/test/regress/expected/multi_create_table_constraints.out

```diff
@@ -59,6 +79,16 @@ CREATE TABLE ex_on_non_part_col
 SELECT create_distributed_table('ex_on_non_part_col', 'partition_col', 'hash');
 ERROR:  cannot create constraint on "ex_on_non_part_col"
 DETAIL:  Distributed relations cannot have UNIQUE, EXCLUDE, or PRIMARY KEY constraints that do not include the partition column (with an equality operator if EXCLUDE).
+-- check that we can disable the constraint check
```

Comment:
might be nice to add a few more tests where define:

* unique on multiple non-distribution columns
* exclusion constr. on multiple non-distribution columns
* unique on multiple non-distribution columns + distribution column
* exclusion constr. on multiple non-distribution columns + distribution column